layouts/shortcodes/blocks/feature.html

```diff
@@ -2,10 +2,17 @@
 {{ $url_text := .Get "url_text" -}}
 <div class="col-lg-4 mb-5 mb-lg-0 text-center">
 <div class="mb-4 h1">
+{{ with .Get "url" }}<a class="text-white" target="_blank" href="{{ . }}">
+{{ end }}
   <i class="{{ if not (or (hasPrefix $icon "fas ") (hasPrefix $icon "fab ")) }}fas {{ end }}{{ $icon }}"></i>
+{{ if .Get "url" }}</a>{{ end }}
 </div>
 <h4 class="h3">
-  {{- .Get "title" | markdownify -}}
+{{ if .Get "url" }}
+  <p><a target="_blank" href="{{ .Get "url" }}">{{ .Get "title" | markdownify }}</a></p>
```

Comment:
Minor nit: If you use `with` here, you can simplify `{{ .Get "url" }}` to  `{{ . }}`

Reply:
Resolved in commit with id [1298ee4](https://github.com/google/docsy/pull/1396/commits/1298ee494bef873792a45d0c01b5e2bcaa7a18f8)

Reply:
Resolved